documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,90 @@
+   2. The competition nickname and ID must resemble the competition name.
+      1. The competition nickname and ID should only be modified if the competition name is too long (more than 32 characters).
+2. Location(s)
+   1. A location consists of the country, city, and venue.
+   2. The location should also include subregions of the country that are common to include with addresses (e.g. the state a U.S. city is located in).
+3. WCA Delegate(s)
+4. Organizer(s)
+   1. Organizers can be specific persons and/or organizations.
+5. Competitor limit
+   1. The competitor limit can be on either a "first come first serve" basis or based upon qualification results or rankings in the WCA world rankings of a previously announced calendar date.
+      1. Qualification for an event must be based only on results from the same event.
+   2. If the competitor limit for a competition or an event is reached before the competition, the organization team must not approve any additional registrations for the competition or the event.
+   3. The competitor limit is per competition. Per day competitor limits are not allowed.
+6. Registration requirements
+   1. The registration requirements must be clear so that any competitor from anywhere in the world knows how to and has the ability to register.
+      1. The registration requirements must be available in English.
+      2. In addition to the English version, the registration requirements can be available in any other language and should be available in the local language.
+   2. The registration period(s) must be a clearly defined time range with a start and end time and date.
+      1. The first registration period must have a start time and date at least 48 hours after the competition is announced.
+      2. If there is more than one registration period, the organization team must announce each additional registration period at least 48 hours before it starts.
+      3. Different registration periods may have different registration requirements (e.g. restricting which events a competitor can register for).
+   3. Different locations for multi-location competitions may have different registration requirements (e.g. competitor limit).
+7. Events
+   1. Each event must have a planned number of rounds.
+      1. Each round of each event must have a planned format and time limit.
+      2. A round of an event must have planned advancement conditions if there is a subsequent round of the event.
+      3. A round may have a cutoff. A round with a cutoff is a combined round, with the formats as per [WCA Regulation 9b)](wca{regulations/#9b}).
+   2. Only results from events that appear on the competition website in the Events tab and Schedule will be recognized.
+      1. Other events may only be held unofficially.
+8. Schedule
+   1. The schedule must be published on the WCA website using the WCA schedule tool.
+9. Optional requirements
```

Comment:
```suggestion
9. Championship type
    1. Championship types are for national, continental, and world championships.
    2. There should be no other competition of the same championship type in the same year.
    3. The championship has most of the officially recognized WCA events.
    4. The championship awards titles and awards to the winners within the respected region. 
    5. The organization team should involve the persons expected to be involved in the championship, otherwise those persons should be notified and satisfied with the organization team.
10. Optional requirements
```

I believe these are the 5 points of interest, I'm open ears to other points.
I feel we maybe should mention the naming policy with championships, but that would be redundant with 1.1.

5. is a little weirdly worded, if others have suggestions, I'm also all in :)

Reply:
Maybe change 5. to "The organization team should consist of the Delegates of the respective region and the respective regional organization, otherwise those persons should be notified and satisfied with the members of the organization team" since I think that would cover the "expected persons". It doesn't really work for continental and world championships. Maybe add something about where a bidding process has taken place?

Reply:
Here is the new and improved wording:

```suggestion
9. Championship type
    1. A championship type is assigned to competitions that are national, continental, or world championships.
        1. A competition may have multiple championship types.
        2. An additional selection process is conducted by the WCA Board for organizing continental and world championships.
    2. A championship type for a region must only be assigned to one competition in a year.
    3. The competition should have most of the officially recognized WCA events.
    4. The competition should recognize champion titles and award them and their respective prizes to the winners in the region.
    5. The organization team of a national championship should involve all the Delegates of the respective country.
10. Optional requirements
```

Reply:
I think the wording is good enough now. @thewca/board do you agree or is there anything else to change?

Reply:
I edited a bit the improved wording. The only actual changes apart from styling are in points 3 ("most of" to "all", as the sentence uses a should) and 4 ("should" to "must"). I'll commit these changes soon if no one opposes!

Reply:
Keep in mind that those points use "should", so reasons like the ones you describe are good enough to not follow the recommendation in such cases.

Reply:
For 3 I agree, but what's the rationale behind 5? In some countries, the distances could be large, so why even suggest to drag them over there just for the sake of what? Also Bob's example is another thing. When we use "should" rules, we still try to aim to them, while having 50+ Delegates as an aim is not reasonable.

Reply:
I think the point of #5 is the opposite side. You shouldn’t have a foreign delegate conduct the national championship with a country of delegates.

The original read something like the persons expected to be involved in the organization teams hoild be.

Reply:
Do you guys have any alternative wording in mind?

Reply:
What about something like "at the discretion of the Senior Delegate"?

Reply:
I like that.

Reply:
That sounds good to me too!

Reply:
For point 4, perhaps:
`Championships must recognize and award champion titles and their respective prizes to the winners in the region.`
It's a weird sentence to de-clunk.

Reply:
I'm fine with that suggestion. Others?

Reply:
Some new possible additions:

`The competition must be located in the region where the championship type is selected.`
Ie. You can have the European championship in America, or the Portuguese championship in Spain.

And then this one may require more discussion, and definitely needs better wording:
`A championship type cannot be on the same date as a conflicting championship type at a different competition.`
This would mean that Canada can't hold the Canadian Nationals on the same date as worlds or Euros and UK championship can't happen on the same weekend. This idea comes from the fact that in 2017 the Canadian nationals was held on the same weekend as worlds because "so someone other than Antoine and Bill can win". The last part of the sentence "different competitions" Is to ensure WC 2013 and 2019 can have the respective national championship at the same competition.

Reply:
For the first one, sure, that seems obvious.
The second one, I don't necessarily agree. If anything, maybe it can be a "should not." Otherwise, this also prevents WC 2021 from being announced on the same weekend of an already announced national championship. I think this point is too restrictive.

Reply:
But with a “should not” it would be reasonable to reject the national championship type on the same day of worlds with the worlds date being know, correct?

Reply:
In certain cases, yes. The further and poorer thr country is, though, the less reasonable it may be. For example, I think it's fine for Venezuela to have its national championship on the same day of Worlds in Australia, but New Zealand or United States would be less appropriate.

Reply:
So you’d also think the Canada situation described above is okay? Cause I certainly think it’s very cheeky.

Reply:
I think it should be avoided. Competitors from Canada are more likely to be able to afford the trip to Worlds. But with Euro, I think it's probably okay. The competitor would be making a choice to forfeit a national title opportunity to attend a competition in which they are not eligible for one. Should it be avoided? Yes. Should it be prohibited? No.

Reply:
Ah that's why I said conflicting. A competitor from Canada doesn't qualify as being able to be a european champion, so I wouldn't consider it conflicting. Whereas in the Canada/Worlds scenario the competitors have to choose which championship they'd rather be a part of. This feels very unfair to many competitors who now don't have a chance to even participate in the national championship every other year (assuming Canada will hold its nationals the same weekend as worlds from now on). This can be even worse for continents with championships. Let's say Hungary has a national championship that is always on the weekend of Euros or Worlds; then the top competitors from Hungary may never get the chance to participate in their own national championship since they are competing for the Euro/World champion titles.

Reply:
I'm adding both points on the new version I'm working on.